src/Microsoft.Dnx.Compilation.CSharp/RoslynProjectReference.cs

```diff
@@ -151,7 +202,69 @@ public void EmitReferenceAssembly(Stream stream)
 
         public DiagnosticResult EmitAssembly(string outputPath)
         {
-            IList<ResourceDescription> resources = CompilationContext.Resources;
+            IEnumerable<ResourceDescription> resources = Enumerable.Empty<ResourceDescription>();
+
+            foreach (var resourceGrouping in CompilationContext.Resources
+                .GroupBy(ResourcesForCulture.GetResourceCultureName, StringComparer.OrdinalIgnoreCase))
+            {
+                if (resourceGrouping.Key == string.Empty)
+                {
+                    resources = resourceGrouping
```

Comment:
In line 301 for `CompilationContext.Compilation.Emit` call. `resourceGrouping.Key` is empty if you have a `resx` that is not locale specific like `test.resx`